### PR TITLE
fix: Restore ElevatedView support for WinAppSDK

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/11011

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

This changes explicitly includes PRI files and related assets in the referencing app, so that `ElevatedView` works properly.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: build/uno.winui.winappsdk.targets [hunk 1]
```diff
@@ -37,4 +37,31 @@
...
+						OutputGroup="CopyLocalFilesOutputGroup" />
+
+			<_UnoUIToolkitAssets Include="$(PkgUno_WinUI)/lib/net5.0-windows10.0.18362.0/Uno.UI.Toolkit/**"/>
+
+			<None
+				Include="@(_UnoUIToolkitAssets)"
+				TargetPath="Uno.UI.Toolkit/%(RecursiveDir)%(Filename)%(Extension)"
+				CopyToOutputDirectory="PreserveNewest" />
+
+			<_UnoUIToolkitAssets Remove="@(_UnoUIToolkitAssets)" />
```
Comment (Youssef1313) [inline build/uno.winui.winappsdk.targets:62]: Curious why this is needed

**Review by MartinZikmund — APPROVED** [review]

File: build/uno.winui.winappsdk.targets (hunk 1 above)
Comment (jeromelaban) [reply build/uno.winui.winappsdk.targets:62]: It's a memory cleanup technique, commonly used inside the SDK's own targets.

**Review by agneszitte-nventive — APPROVED** [review]

**Comment by MartinZikmund** [comment]
Should this be backported to 4.7?

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/4.7

**Pull request merged**